python/pyarrow/_s3fs.pyx

```diff
@@ -44,13 +44,22 @@ def initialize_s3(S3LogLevel log_level=S3LogLevel.Fatal):
     ----------
     log_level : S3LogLevel
         level of logging
+
+    Examples
+    --------
+    >>> fs.initialize_s3(fs.S3LogLevel(5)) # doctest: +SKIP
```

Comment:
It would be much better to showcase a symbolic constant, for example:
```suggestion
    >>> fs.initialize_s3(fs.S3LogLevel.Error) # doctest: +SKIP
```